app/javascript/components/Article.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default ({ children }) => {
+	return (
```

Comment:
Can we down-size the tabbing on this (and other similar files) from 8 spaces to 2?